policyengine_us/variables/gov/states/az/hhs/tanf/az_hhs_tanf_earned_income_care_expense_disregard.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class az_hhs_tanf_earned_income_care_expense_disregard(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Arizona Cash Assistance care expense earned income disregard"
+    definition_period = MONTH
+    defined_for = StateCode.AZ
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.hhs.tanf.eligibility.income.earned.care_expenses
+        # Get the age of the child(children) and the disabled adult(s)
+        person = spm_unit.members
+        age = person("age", period) * MONTHS_IN_YEAR
+        # Get the childcare and disabled adult care expenses
+        care_expenses = spm_unit("childcare_expenses", period)
```

Comment:
@PavelMakarchuk do we have a variable that includes adult care? like `care_expenses`?

Reply:
@liwenxin491 lets do:
```suggestion
        care_expenses = add(spm_unit, period, ["childcare_expenses", "care_expenses"])
```

Reply:
i thought `care_expenses` included `childcare_expenses`

Reply:
no, its is a sum of `pre_subsidy_care_expenses`

Reply:
but isn't that intended to encompass childcare? where is it used today?

Reply:
I believe this was supposed to be a standalone var for adult /elderly care expenses as it has its own `pre_subsidy_care_expenses` but we should consolidate the entire logic to make it clear